juju/juju
# Regenerate ec2 instance type data.

## Description of change

This now includes a new region (ap-northeast-3), and new machine types
x1e, c5, m5, p3.

Note that index.json is now up to 210MB, but it does feel like we should do something like this as a regular release step.

The diff is is a bit ugly because of the new region, though if you do `git diff --patience` or `git show HEAD --patience` you can see the change very clearly.

## QA steps

`$ juju bootstrap aws/ap-northeast-3` does not currently work. We need to update our clouds definition as well as the ec2 type information.

However:
`$ juju deploy ubuntu --constraints instance-type=x1e.xlarge`
On regions that support it. (ap-northeast-1, ap-southeast-2, eu-west-1, us-east-1, us-west-2, appear to be the only ones that support x1e.*)

## Documentation changes

None

## Bug reference

[lp:1754735](https://bugs.launchpad.net/juju/+bug/1754735)
